docs/hiring-guide/portfolios.rst

```diff
@@ -0,0 +1,124 @@
+Maintaining a portfolio
+-----------------------
+
+Most job searches require that technical writers produce one or more samples of their writing.
+Managers and other interviewers rely on such samples to determine the abilities of the applicant.
+Because nearly every job posting expects a writer to provide a writing sample, a tech writer on the job hunt should have one or more samples ready.
+
+What to include in a portfolio
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A portfolio consists of a set of the writer's best samples of their work.
+What "best samples" means is entirely subjective on your part.
+When building out a portfolio, include samples of writing that
+
+- you are exceptionally proud of
+- demonstrate a particular skill
+- validates domain knowledge in a particular area
```

Comment:
```suggestion
- validate domain knowledge in a particular area
```

Reply:
I believe the noun these verbs are referring to is "samples" which is plural so you need the plural version of the verbs.

Reply:
nice catch, thanks!